plugins/LadspaEffect/LadspaControlDialog.cpp

```diff
@@ -70,10 +70,9 @@ LadspaControlDialog::LadspaControlDialog( LadspaControls * _ctl ) :
 void LadspaControlDialog::updateEffectView( LadspaControls * _ctl )
 {
 	QList<QGroupBox *> list = findChildren<QGroupBox *>();
-	for( QList<QGroupBox *>::iterator it = list.begin(); it != list.end();
-									++it )
+	for(auto & it : list)
```

Comment:
When the container isn't a plural word, it seems to reuse the iterator name from before. This is misleading, as the variable is no longer an iterator, but the item itself.